packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx

```diff
@@ -1,3 +1,7 @@
+import React, { forwardRef, useContext, useMemo } from 'react';
+import classNames from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
+import { useTranslation } from 'react-i18next';
```

Comment:
This change just reorganises the imports (note to self to prioritise work on adding https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md to our ESLint config for automatically handling this concern)

Reply:
That's amazing!